README.md

```diff
@@ -2,3 +2,171 @@
+```sh
+$ git diff
+...
+```
+
+You should see new function calls and wrappers placed around relevant code such as http handler functions and sql clients. You'll also see comments added, such as `//dd:startwrap`. These comments are used to identify instrumented code, so that orchestrion can be run multiple times without duplicating instrumentation, and also so orchestrion can remove instrumentation when instructed to with the `-rm` flag.
+
+Now, shut down the services:
+```
+$ docker-compose down
+```
+
+We need to update our go.mod files before we can build the new code. The `Orchestrion` instrumentation requires code from the `github.com/DataDog/orchestrion` library.
+
+```sh
+$ cd services/users
+services/users $ go mod tidy
+go: finding module for package github.com/datadog/orchestrion/instrument
+go: found github.com/datadog/orchestrion/instrument in github.com/datadog/orchestrion v0.1.0
+services/users $ cd ../notes/
+services/notes $ go mod tidy
+go: finding module for package github.com/datadog/orchestrion/instrument
+go: found github.com/datadog/orchestrion/instrument in github.com/datadog/orchestrion v0.1.0
+services/notes $ cd ../../
+$ 
+```
+
+Then rebuild the applications with our newly instrumented code
+```sh 
+$ docker-compose build
+```
+
+And start the services again:
+| :zap: Remember that the DD_API_KEY environment variable needs to be set! |
+|--------------------------------------------------------------------------|
+```sh 
+$ docker-compose up -d
+```
+
+Now, going to the application home page [http://localhost:8080](http://localhost:8080) will show us the same page, but looking at our Traces Page in the Datadog app [https://app.datadoghq.com/apm/traces](https://app.datadoghq.com/apm/traces) should show some new traces appear!
+
+![First Trace](doc/trace1.png)
+
+Click through the various spans and tabs. For instance, by clicking the `sqlite3.query` span, we can see the SQL query and other info.
+![SQLite3 Trace Data](doc/sqlite-info.png)
+
+
+## More Traces
+Now that we have instrumentation, let's generate some more interesting traces. From the browser, click on one of the users' IDs to go to their notes page.
+
+Navigating to that page should generate another trace, which looks like this:
+![Notes Page](doc/notes-trace.png)
+
+Here we can see that the `users` service received an http request. It then sent an outgoing http request to the `notes` service, requesting `/notes?userid=2` in this case (this will depend on which user's page you are viewing).
+![Outgoing](doc/notes-trace2.png)
+
+The `notes` service then executes a SQL query:
+![Notes SQL Query](doc/notes-sql.png)
+
+
+Try adding a new note. When you do that, you should see a new trace appear.
+![New Note Trace](doc/notes-trace3.png)
+This trace appears with type `POST` and status code `302` because the `users` service returns a redirect to the browser after the submission succeeds.
```

Comment:
```suggestion
2. Add a new note. When you do, you should see a new trace appear.

![New Note Trace](doc/notes-trace3.png)

This trace appears with type `POST` and status code `302` because the `users` service returns a redirect to the browser after the submission succeeds.
```